kong/init.lua

```diff
@@ -1152,4 +1152,22 @@ function Kong.admin_header_filter()
 end
 
 
+function Kong.serve_status_api(options)
+  kong_global.set_phase(kong, PHASES.admin_api)
+
+  local options = options or {}
```

Comment:
This argument is never given to `serve_status_api`, let's not introduce it only to remove it later on. A subsequent commit can remove the one from `admin_content` later on.